docs/adrs/20210729-stage-file-proxy.md

```diff
@@ -0,0 +1,55 @@
+# Stage File Proxy for static files
+
+Lullabot often works on projects that have large numbers of static file assets in production, such as images or PDFs. Keeping a complete copy of those files for each environment is wasteful, since developers tend to only look at a small portion of a site locally. A solution is needed to reduce disk use and the time it takes to update an environment.
+
+---
+
+date: 2021-10-26
+
+status: accepted
+
+tags:
+  - devops
+  - locals
+  - drupal
+
+deciders:
+  - Andrew Berry
+  - Darren Petersen
+  - Marcos Cano
+  - Chris Albrecht
+
```

Comment:
@greggmarshall as you mention in your notes, let's change the deciders according to our org. 

Can I suggest that you add the following deciders?

- Jordi Ros-Giralt
- Javier Prieto
- Gregg Marshall (if you think it's ok)

Thanks.